test/conformance/ptp/ptp.go

```diff
@@ -222,27 +222,14 @@ var _ = Describe("[ptp]", func() {
 
 				By("Getting the NIC details of all the PTP enabled interfaces")
 
-				var mapping = make(map[string]string)
-				for _, pod := range ptpRunningPods {
-					mapping = getNICInfo(pod)
-					Expect(mapping).ShouldNot(BeEmpty())
-				}
-
-				By("Getting the interface details of the PTP config")
+				ptpInterfacesList := fullConfig.L2Config.GetPtpIfList()
 
-				ptpConfig := testconfig.GlobalConfig
-				if ptpConfig.DiscoveredGrandMasterPtpConfig != nil {
-					printInterface(ptpv1.PtpConfig(*ptpConfig.DiscoveredGrandMasterPtpConfig), mapping)
-				}
-				if ptpConfig.DiscoveredClockUnderTestPtpConfig != nil {
-					printInterface(ptpv1.PtpConfig(*ptpConfig.DiscoveredClockUnderTestPtpConfig), mapping)
-				}
-				if ptpConfig.DiscoveredClockUnderTestSecondaryPtpConfig != nil {
-					printInterface(ptpv1.PtpConfig(*ptpConfig.DiscoveredClockUnderTestSecondaryPtpConfig), mapping)
+				for _, ptpInterface := range ptpInterfacesList {
```

Comment:
Only writing to the log not to teh report ?

Reply:
We decided to merge this PR this way. And then work on the reporting using v1 in some way as a temporary solution.